it/rules.md

```diff
@@ -0,0 +1,60 @@
+
+#### _Qualsiasi violazione delle regole in questa sezione risulterà in un **silence**._
+
+* Non **spammare** in chat.
+* Non condividere **robe illegali/warez/virus** in chat.
+* Non **offendere** gli altri giocatori nella chat.
+* Non abusare delle **imprecazioni.**
+* Non fare nessun tipo di **accusa di cheating** (Usa il canale #cheaterreporting su Discord o denuncia la persona che sospetti ad un Community Manager)
+
+
+<h3><i class="user icon"></i> Regole pagina utente/avatar</h3>
+
+#### _Qualsiasi violazione delle regole in questa sezione risulterà in un **silence ed un reset della pagina utente/avatar**._
+
+* I contenuti degli avatar/pagina utente **non adatti all'ambiente di lavoro ed offensivi** non sono permessi.
+* Non mettere link a **cose illegali**/porno/warez/virus sulla pagina utente.
+* Non usare **badge personalizzati offensivi** e non **pretendere di essere qualcun altro con il tuo badge personalizzato**. Se abusi del sistema dei badge, verrai silenziato e non sarai più in grado di modificare il tuo badge personalizzato.
+
+<h3><i class="comment icon"></i> Regole Discord</h3>
+
+#### _Qualsiasi violazione delle regole in questo server discord risulterà in un <u>silence/espulzione</u> o, nel peggior caso, un **ban**._
+
+* Comportati **decentemente**.
+* Non spammare nel canale **#general**, usa invece **#random**.
+* Non condividere CP (Child Porn) o altre **cose illegali/warez/virus**.
+* Non far infuriare gli amministratori.
+* Non fare nessun tipo di **accusa di cheating** (Usa il canale #cheaterreporting su Discord o denuncia la persona che sospetti ad un Community Manager)
+
+<h3><i class="file text outline icon"></i> Politica cheat/multiaccount</h3>
+
+#### _Questo è il modo in cui gestiamo i cheater e i multiaccounter su Ripple._
+
+* Non ci interessa se sei stato un cheater nel passato, fallo su osu! su nosue o dove cavolo ti pare. 
+Ciò nonostante, **siamo CONTRARI al cheating**, e agiremo immediatamente su qualsiasi punteggio hackerato.
+* I multiaccount **NON** sono permessi, con la **SOLA ECCEZIONE DEI BOT PER LA CHAT**.  Se ne stai sviluppando uno e hai bisogno di un account per esso,  manda una mail al [supporto](mailto:support@ripple.moe) (o ad uno [sviluppatore](mailto:howl@ripple.moe))
+* Ci riserviamo il diritto di cambiare queste regole in base a come riteniamo.
+
+<h3><i class="list layout icon"></i> Varie</h3>
+
+* [Leggi le due regole seguenti con questa canzone in sottofondo.](https://www.youtube.com/watch?v=OBQE_TNI7zw)
+* Puoi cambiare la bandiera del tuo profilo mandando una mail al [supporto](mailto:support@ripple.moe), e mandando una foto di un documento valido ed un post-it o un pezzo di carta con sopra il tuo nome di Ripple. I documenti verranno scrupolosamente visionati ~~dai nostri ispettori di confine~~.
+* Se provi a prendere in giro il team del supporto (ad esempio fornendo falsi documenti), ~~ti rispediamo a Kolechia~~ verrai bannato. Semplicemente così.
```

Comment:
Kolechia is a Papers please country, so it should be `Ti rispediamo in Kolechia`